src/styles/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+
+  &__star {
+    content: '';
+    width: 16px;
+    height: 16px;
+
+    @include no-active-stars;
+
+    &:nth-of-type(1) {
+      margin-right: $commonMargin;
+    }
+
+    &:nth-last-of-type(1) {
+      margin-left: $commonMargin;
+    }
+
+    &:not(&:nth-of-type(1), &:nth-last-of-type(1)) {
+      margin-inline: $commonMargin;
+    }
```

Comment:
It's better to simplify it, just add margin in one direction and remove it from last element by using last-child pseudoclass

Reply:
Thanks, solved.